src/applications/healthcare/questionnaire/containers/ConfirmationPage.jsx

```diff
@@ -1,42 +1,41 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import moment from 'moment';
 import { connect } from 'react-redux';
-import Scroll from 'react-scroll';
 
-import { focusElement } from 'platform/utilities/ui';
-
-const scroller = Scroll.scroller;
-const scrollToTop = () => {
-  scroller.scrollTo('topScrollElement', {
-    duration: 500,
-    delay: 0,
-    smooth: true,
-  });
-};
+import Telephone from '@department-of-veterans-affairs/formation-react/Telephone';
 
 const ConfirmationPage = props => {
-  useEffect(() => {
-    focusElement('.schemaform-title > h1');
-    scrollToTop();
-  }, []);
+  // useEffect(() => {
+  //   focusElement('.schemaform-title > h1');
+  //   scrollToTop();
+  // }, []);
```

Comment:
Not sure about front-end style guides but should we be leaving commented out code in here?